From: Beren Minor <beren.minor+debian@gmail.com>
Date: Sun, 5 Feb 2012 20:41:37 +0100
Subject: DEBIAN: Add GemRB version number to gemrb_core library.

---
 gemrb/core/CMakeLists.txt |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/gemrb/core/CMakeLists.txt b/gemrb/core/CMakeLists.txt
index d57fac2..44653b7 100644
--- a/gemrb/core/CMakeLists.txt
+++ b/gemrb/core/CMakeLists.txt
@@ -149,9 +149,12 @@ else (STATIC_LINK)
 	ENDIF(WIN32)
 endif (STATIC_LINK)
 
+STRING(REGEX REPLACE "\\..*$" "" GEMRB_SOVERSION "${GEMRB_VERSION}")
 SET_TARGET_PROPERTIES(gemrb_core PROPERTIES
 	COMPILE_DEFINITIONS
 	"PLUGINDIR=\"${PLUGIN_DIR}\";DATADIR=\"${DATA_DIR}\";SYSCONFDIR=\"${SYSCONF_DIR}\""
+	VERSION ${GEMRB_VERSION}
+	SOVERSION ${GEMRB_SOVERSION}
 	)
 
 IF (APPLE)
